west.yml

```diff
@@ -21,9 +21,10 @@ manifest:
           - hal_nxp
           - littlefs
           - loramac-node
-          - hal_st
```

Comment:
Note github diff doesn't show that it's present above. This is just deleting a duplicate entry